app/mapping.py

```diff
@@ -1,25 +1,38 @@
 from flask import json
 
 
-with open("mappings/services.json") as services:
-    SERVICES_MAPPING = json.load(services)
+SERVICES_MAPPING_FILE_SPEC = "mappings/services.json"
 
-FILTER_FIELDS = sorted(
-    field.replace('filter_', '')
-    for field in SERVICES_MAPPING['mappings']['services']['properties'].keys()
-    if field.startswith('filter_')
-)
+_services = None
 
-TEXT_FIELDS = sorted(
-    field
-    for field in SERVICES_MAPPING['mappings']['services']['properties'].keys()
-    if not field.startswith('filter_')
-)
 
-AGGREGATABLE_FIELDS = sorted(
-    k
-    for k, v in SERVICES_MAPPING['mappings']['services']['properties'].items()
-    if v.get('fields', {}).get('raw', False)
-)
+class Mapping(object):
+    def __init__(self, mapping_definition, mapping_type):
```

Comment:
Unfortunately that particular piece of tediousness came from Elasticsearch themselves :/ https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html